s3.go

```diff
@@ -110,6 +110,14 @@ func (e *Client) PutObject(ctx context.Context, bucket string, key string, reade
 	return e.client.PutObject(ctx, e.options.Prefix+bucket, key, reader, objectSize, e.putOpts)
 }
 
+func (e *Client) ListObjects(ctx context.Context, bucket string, dir string) <-chan minio.ObjectInfo {
+	e.logger.Debug().Msgf("listing objects in bucket '%s' (prefix '%s') for dir %v", bucket, e.options.Prefix, dir)
```

Comment:
Renaming `dir` to `subprefix`? Not sure what the prefix is being used for since each user gets their own bucket anyways. Is this for the filename?

Reply:
Oh yes! The prefix here is just the S3 terminology prefix, so more or less the "directory" we want to list files in. You're def. right though, staying consistent here matters

Reply:
Resolved in 32ac5777168678a420e67c8207c9466f1fd4e29d